cfgov/ask_cfpb/scripts/migrate_knowledgebase.py

```diff
@@ -35,6 +36,57 @@
 }
 
 
+def unwrap_soup(soup):
+    soup.html.unwrap()
+    soup.body.unwrap()
+    return unicode(soup)
+
+
+def convert_divs_to_asides(answer_text):
+    soup = bs(answer_text, 'lxml')
+    for div in soup.findAll('div', {'class': 'answer-module'}):
+        div.name = 'aside'
+    return unwrap_soup(soup)
+
+
+def clean_orphan_tips(answer_text):
+    headings = ['TIP', 'WARNING', 'NOTE']
+    soup = bs(answer_text.replace('<br/><br/>', '</p><p>'), 'lxml')
+    h4s = soup('h4')
+    for hed in h4s:
+        if hed.parent.name == 'p' and hed.text.strip(':').upper in headings:
+            wrapper = hed.parent.wrap(
+                soup.new_tag('aside'))
+            wrapper['class'] = 'answer-module'
+    strongs = soup('strong')
+    for strong in strongs:
+        # clean out spans
+        if strong('span'):
+            for span in strong('span'):
+                span.unwrap()
+        # turn strong tips into h4 and wrap in aside
+        if strong.text.strip(':').upper() in headings:
+            strong.name = 'h4'
+            if strong.parent.name == 'p':
+                wrapper = strong.parent.wrap(
+                    soup.new_tag('aside'))
+                wrapper['class'] = 'answer-module'
+                if ':' in wrapper.contents:
+                    i = wrapper.contents.index(':')
+                    del(wrapper.contents[i])
+    return unwrap_soup(soup)
+
+
+def fix_tips(answer_text):
+    try:
+        clean1 = convert_divs_to_asides(answer_text)
+        clean2 = clean_orphan_tips(clean1)
+    except Exception:
```

Comment:
I realize this script is intended to be run once on a known dataset, but `Exception` here feels overly broad. What can `clean_orphan_tips` raise that we'd want to catch? Will `convert_divs_to_asides` raise anything? It seems like we expect an exception to indicate that there are no tips, is that correct?